bin/encore.js

```diff
@@ -60,7 +60,7 @@ if (runtimeConfig.useDevServer) {
 }
 
 function showUsageInstructions() {
-    const validCommands = ['dev', 'production', 'dev-server'];
+    const validCommands = ['dev', 'prod', 'production', 'dev-server'];
```

Comment:
Is it an idea to also add `development` for consistency? So we could have dev, development, prod and production? Or is that unnecessary?

Reply:
In my case I would remove "production", I don't think it's useful to add longer commands.

Reply:
Indeed, I think we share the same thoughts. Although I'd like to see short commands only, I don't think removing one is a good idea in terms of backwards compatibility issues.

Reply:
What about change documentation with short names (to make people use the new aliases), and keep long names in code for compatibility ? I will do same PR for Symfony environment variable then.

Reply:
I think that's not a bad idea at all, we might need to ping in @stof and @weaverryan for this matter, maybe they can share their opinions.